### ISSUE TITLE
PMO: Going directly to /restaurants in production causes errors

### ISSUE THREAD

**Issue by matthewp** [donejs/donejs#479]
This is coming from can-wait

```
can-serve starting on http://localhost:8080
Potentially unhandled rejection [1] TypeError: Cannot read property '__timeoutId'
of undefined
    at C:\Users\Matthew\dev\place-my-order\node_modules\can-wait\can-wait.js:95:32

    at f [as _set] (file:C:/Users/Matthew/dev/place-my-order/dist/bundles/place-my
-order/index.js:101:4860)
    at t.simpleExtend.set (file:C:/Users/Matthew/dev/place-my-order/dist/bundles/p
lace-my-order/index.js:51:3001)
    at p (file:C:/Users/Matthew/dev/place-my-order/dist/bundles/place-my-order/ind
ex.js:53:264)
    at o (file:C:/Users/Matthew/dev/place-my-order/dist/bundles/place-my-order/ind
ex.js:101:6226)
    at s (file:C:/Users/Matthew/dev/place-my-order/dist/bundles/place-my-order/ind
ex.js:101:8227)
    at f (file:C:/Users/Matthew/dev/place-my-order/dist/bundles/place-my-order/ind
ex.js:101:7873)
    at d (file:C:/Users/Matthew/dev/place-my-order/dist/bundles/place-my-order/ind
ex.js:101:8011)
    at i.data (file:C:/Users/Matthew/dev/place-my-order/dist/bundles/place-my-orde
r/index.js:101:1240)
    at o.<anonymous> (file:C:/Users/Matthew/dev/place-my-order/dist/bundles/place-
my-order/index.js:103:2409)
^CTerminate batch job (Y/N)? Y
PS C:\Users\Matthew\dev\place-my-order>

```

**Comment by matthewp** [comment]
This is the underlying issue: https://github.com/canjs/can-wait/issues/32

Confirmed that fixing this will fix the issue.

**Comment by matthewp** [comment]
This is fixed in can-wait 0.2.7.
